pkg/reconciler/managed/reconciler.go

```diff
@@ -545,10 +545,10 @@ func (r *Reconciler) Reconcile(_ context.Context, req reconcile.Request) (reconc
 		// If this is the first time we encounter this issue we'll be requeued
 		// implicitly when we update our status with the new error condition.
 		// If not, we want to try again after a short wait.
-		log.Debug("Cannot initialize managed resource", "error", err, "requeue-after", time.Now().Add(r.shortWait))
+		log.Debug("Cannot initialize managed resource", "error", err)
```

Comment:
We don't need to bring this into scope for this PR, but it would be great if we could eventually make our backoff state inspectable such that we could keep something like the `requeue-after` context. I'd feel more comfortable with longer backoffs (e.g. multiple minutes) if we could provide users with context as to how long our backoff was and why.

Reply:
Yeah unfortunately we don't have access to the rate limiting interface within the reconciler (it is set at the controller level) and the controller-runtime default actually doesn't allow any sort of "peek" method that doesn't also increment the failure count used for backoff. This was part of my motivation for having a "layer 2" rate limiter because we could:
1. Set a default without folks having to explicitly pass it in to the controller.
2. Decouple the external API backoff from the k8s API server one.
3. Do things like log what the backoff value is currently at.

Reply:
I'd prefer to tackle this (someday) at the controller-runtime level if possible. It would be nice if, for example, the `controller.Manager` had a method we could call to ask what the soonest time we'd be requeued at was.